### PR TITLE
Restore 2.4.6 GraphQL schema

### PR DESCRIPTION
**Pull request by jhadobe**
This PR adds the older GraphQL schema for 2.4.6, following a request from the field.

I have removed the unused beta structure.

[Staging](https://adobedocs.github.io/commerce-webapi/graphql/reference/2.4.6/)

For a quick sanity check, i've confirmed the following are in the `2.4.7` version but NOT in the `2.4.6` version:

- clearCart mutation
- recaptchaV3Config queries

### CODE REVIEW

File: src/data/navigation/sections/graphql.js [hunk 1]
```diff
@@ -1317,6 +1317,10 @@ module.exports = [
     title: "Reference (2.4.7)",
     path: "/graphql/reference",
   },
+  {
```
Comment (jeff-matthews) [inline src/data/navigation/sections/graphql.js:1320]: Would prefer to have this nested under "Reference (2.4.7)" with "2.4.6" as the heading.

File: src/pages/graphql/reference/legacy/index.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
```
Comment (jeff-matthews) [inline src/pages/graphql/reference/legacy/index.md:1]: I don't think "legacy" is quite right. I think using the version number would be cleaner.

**Review by jeff-matthews — COMMENTED** [review]
I left a couple of comments in the changed files, but I also think reusing the beta configs and static directories could cause problems if/when we start publishing the next beta schema in October.

**Comment by jhadobe** [comment]
@jeff-matthews ready for re-review, the staging environment seems to have cached the old menu structure, may need to view in incognito.

File: src/data/navigation/sections/graphql.js [hunk 3]
```diff
@@ -1316,6 +1316,12 @@ module.exports = [
   {
     title: "Reference (2.4.7)",
     path: "/graphql/reference",
+    pages: [
+      {
+        title: "Reference (2.4.6)",
```
Comment (jeff-matthews) [inline src/data/navigation/sections/graphql.js:1321]:
```suggestion
        title: "2.4.6",
```

**Review by jeff-matthews — COMMENTED** [review]
Nice @jhadobe. Just a small request on the TOC.

File: src/pages/graphql/reference/2.4.6/index.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: GraphQL API legacy reference
```
Comment (jeff-matthews) [inline src/pages/graphql/reference/2.4.6/index.md:2]:
```suggestion
title: GraphQL API reference (2.4.6)
```

**Review by jeff-matthews — COMMENTED** [review]
Sorry, I forgot to comment on the index.md file for 2.4.6.

**Review by jeff-matthews — APPROVED** [review]

**Pull request merged**